tcp/README.md

```diff
@@ -9,18 +9,16 @@ TCP server for JSON-RPC 2.0.
 
 ```
 [dependencies]
-jsonrpc-core = "6.0"
-jsonrpc-tcp-server = "6.0"
+jsonrpc-tcp-server = { git = "https://github.com/paritytech/jsonrpc" }
```

Comment:
Is it okay to fetch git dependency everywhere?

Reply:
yup!